client/components/TestRun/TestRun.css

```diff
@@ -158,6 +158,27 @@ button.test-navigator-toggle:focus {
     left: 4px;
 }
 
+.test-name-wrapper.bot-running .progress-indicator {
+    background: #d2d5d9;
```

Comment:
not sure how strict this requirement is:

"in progress (blue with green circle around)"

Reply:
I went with the colors in the image from issac linked on #820 (assuming the one from the designer was correct)

Reply:
Ah gotcha! Didn't see those